packages/payment-processor/src/index.ts

```diff
@@ -12,6 +12,7 @@ export * from './payment/swap-any-to-erc20';
 export * from './payment/swap-erc20';
 export * from './payment/swap-erc20-fee-proxy';
 export * from './payment/conversion-erc20';
+export * from './payment/erc20-escrow-payment';
```

Comment:
considering the number of functions and the specificities, I would maybe group these exports like this: 
```suggestion
export * as Escrow from './payment/erc20-escrow-payment';
```

so the usage would be 

```ts 
import { Escrow } from "@requestnetwork/payment-processor";

Escrow.approveErc20ForEscrow(...)

```

Reply:
Thanx, I fix.